deploy/podman/okd-configmap.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
```

Comment:
A lot of the values in the original configmap are auto-generated https://github.com/openshift/assisted-service/blob/a9869850e72b002f4e9f89374da2bfd2e60dc43a/hack/generate.sh#L100-L103

Reply:
We also have [automation](https://github.com/openshift-assisted/assisted-installer-deployment/blob/454edbd4fea43c99f6bffc816a3426248a63fa2c/tools/bump_ocp_releases.py#L1) in place to periodically create PRs which bump the z-stream version, you'll probably want to do the same for OKD too

Reply:
Good idea, added a TODO

Reply:
Fixed in 3327bcd1f8ab3a8a83b1d84a0161f3804510374e